app/assets/stylesheets/stories.scss

```diff
@@ -40,6 +40,26 @@
   overflow: auto;
 }
 
+.story-description em {
+  font-style: italic;
+}
+
+.story-description strong {
+  font-weight: bold;
+}
+
+.story-description ul {
+  padding-top: 1em;
+  padding-bottom: 1em;
+  list-style-position: inside;
+  list-style-type: circle;
+}
+
+.modal .story-description p {
```

Comment:
there is already a `.modal .story-description p` selector defined at line 39, we can add the margin there, it already includes the `overflow: auto` property